exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/UpsertTests.kt

```diff
@@ -289,6 +280,10 @@ class UpsertTests : DatabaseTestsBase() {
                 it[item] = "Item B"
                 it[gains] = 200
                 it[losses] = 0
+                // `amount` must be passed explicitly now due to usage of that column inside the custom onUpdate statement
+                // There is an option to call `tester.amount.defaultValueFun?.let { it() }!!`,
+                // it looks ugly but prevents regression on changes in default value
+                it[amount] = 25
```

Comment:
It was one of my biggest concern on upsert statement changes. The main problem is that that column `amount` is used inside `onUpdate` section (and here this section is not autogenerated). Before the changes in the PR it was working, because `amount` with default value was propagated automatically.

Reply:
Thank you, I will add a hint about that in the documentation

About the variant integer("amount").default(25).clientDefault { 25 }, I'd prefer to avoid such a difinition of course, and (not sure, but it can be that) `clientDefault()` may overwrite `default()` because they modify the same fields of column